contracts/staking/interfaces/IValidatorManager.sol

```diff
@@ -32,6 +32,20 @@ struct ValidatorChurnPeriod {
     uint64 churnAmount;
 }
 
+/**
+ * @notice Event emitted when delegator removal is initiated
```

Comment:
Can you make the `@notice` comment more generic? It applies to both delegation registration and removal, but IMO we don't need to mention delegation at all here.